distribution/docs/src/main/resources/_contents/running-contents.adoc

```diff
@@ -322,6 +322,9 @@ Provides a list of environment variables.
 |`spatial`
 |Searches spatially the local Catalog.
 
+|`validate`
+|Runs the specified file against all installed validators.
```

Comment:
It doesn't work on directories, but I definitely like your sentence a lot better.